examples/tool_tests/gowin.c

```diff
@@ -1,10 +1,15 @@
-#pragma PART "GW2AR-18C GW2AR-LV18QN88PC8/I7"
+// @tiagosr: changed part number recognition to pick latest version of a given part
+// it would be good to discuss a standard for device versions. Maybe use a colon for the version?
+// speed grade C8 is equivalent to I7 in this context, and resolves to the expected C8/I7
+// for gw_sh not to complain.
+#pragma PART GW2AR-LV18QN88PC8
```

Comment:
Would it make sense to standardize some means to have device versions in the part string? I reckon from what you mentioned about the Efinix device version selection, it works the opposite way from what Gowin does: there are a lot of Gowin parts that keep the exact same part number but the tool expects you to specify which version to synthesize for, with no defaults.

I made the latest version the default, in the case of the part selection, but if a user wants to select the device version themself, maybe separating by colon would be a good standard (in this case, `GW2AR-LV18QN88PC8:C`)?